From 90bb6b273f91e7296b7dfbe6e193b5aeabfec22d Mon Sep 17 00:00:00 2001
From: Nathan Kinder <nkinder@redhat.com>
Date: Tue, 6 Nov 2012 07:58:58 -0800
Subject: [PATCH 11/16] Ticket #503 - Improve AD version in winsync log
 message

When enabling replication level logging, winsync prints out what
version of Windows/AD it detects.  If it detects win2k3 or later, it
prints out "detected win2k3 peer".  This can be confusing if you are
running a later version, such as win2k8.  The code is really trying
to detect if we can use certain operations that only started being
supported in win2k3.  This patch changes the message to match this
logic by printing "detected win2k3 or later peer".
(cherry picked from commit 98b809ba580530d9418e7bbf857e1ff2b79f75c8)
---
 ldap/servers/plugins/replication/windows_connection.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/windows_connection.c b/ldap/servers/plugins/replication/windows_connection.c
index 0393627..105c205 100644
--- a/ldap/servers/plugins/replication/windows_connection.c
+++ b/ldap/servers/plugins/replication/windows_connection.c
@@ -1333,7 +1333,7 @@ windows_conn_connect(Repl_Connection *conn)
 		if (CONN_IS_WIN2K3 == supports)
 		{
 			windows_private_set_iswin2k3(conn->agmt,1);
-			LDAPDebug( LDAP_DEBUG_REPL, "windows_conn_connect : detected Win2k3 peer\n", 0, 0, 0 );
+			LDAPDebug( LDAP_DEBUG_REPL, "windows_conn_connect : detected Win2k3 or later peer\n", 0, 0, 0 );
 		} else
 		{
 			windows_private_set_iswin2k3(conn->agmt,0);
-- 
1.7.11.7

